GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1928,6 +1934,28 @@ void AddRecentRepositories(Repository repo, string caption)
             }
         }
 
+        private void OpenRepo(string repoPath)
+        {
+            if (ModifierKeys == Keys.Control)
```

Comment:
Please invert `if` and add `return` after `ChangeWorkingDir` call

Reply:
done